### PR TITLE
ukrainian language

### PR DESCRIPTION
**Pull request by maidanskyi**
adds language support for ukrainian language

### CODE REVIEW

**Comment by iamdtang** [comment]
thanks! released in 3.14.0

File: src/lang/ua.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  accepted: ':attribute повиннен бути прийнятий.',
+  alpha: 'Поле :attribute може складатись тільки з літер.',
+  alpha_dash: 'Поле :attribute може складатись тільки з літер, чисел, дефісів та символів підкреслення.',
+  alpha_num: 'Поле :attribute може складатись тільки з літер та чисел.',
+  between: 'Значення поля :attribute повинно знаходитись між :min і :max.',
+  confirmed: 'Поле :attribute не співпадає з підтвердженням.',
+  email: 'Значення поля :attribute повинно бути існуючою електронною адресою.',
```
Comment (vict-shevchenko) [inline src/lang/ua.js:8]: May be "існуючою" -> "дійсною"

File: src/lang/ua.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  numeric: 'Значення поля :attribute повинно бути числом.',
+  present: 'Поле :attribute повинно бути присутнім (але може бути пустим).',
+  required: 'Поле :attribute обов\'язкове для заповнення.',
+  required_if: 'Поле :attribute потрібне у випадку коли значення поля :other рівне :value.',
+  same: 'Значеня поля :attribute повинно співпадати з :same.',
+  size: {
+    numeric: 'Значення поля :attribute повинно бути рівним :size.',
+    string: 'Кількість символів в полі :attribute повинна бути рівною :size.'
+  },
+  url: 'Поле :attribute повинне містити валідний URL.',
```
Comment (vict-shevchenko) [inline src/lang/ua.js:32]: May be "валідний" -> "вірний"

File: src/lang/ua.js (hunk 1 above)
Comment (iamdtang) [reply src/lang/ua.js:8]: I wouldn't know. @Maidanskiy ?

**Comment by vict-shevchenko** [comment]
It seems that also a Gruntfile needs to be modified.

File: src/lang/ua.js (hunk 1 above)
Comment (maidanskyi) [reply src/lang/ua.js:8]: I guess "існуючою" is more accurate

**Comment by iamdtang** [comment]
@vict-shevchenko ah yes. can you make a PR?

File: src/lang/ua.js (hunk 2 above)
Comment (maidanskyi) [reply src/lang/ua.js:32]: the same

File: src/lang/ua.js (hunk 1 above)
Comment (vict-shevchenko) [reply src/lang/ua.js:8]: "існуючою" - means existing,
"дійсною" - means valid.

When you do a validation of email and you have 'sdfasfdasfdfasdf@test.io' this email does not exist but it is valid. We are checking email for validity, not sending real mail to address in order to check for existence.

File: src/lang/ua.js (hunk 2 above)
Comment (vict-shevchenko) [reply src/lang/ua.js:32]: It is not the same. There is no word "валідний" in Ukrainian language, this word was taken from English and written in Ukrainian letters.

"вірний" or "корректний" are much better alternatives.

**Comment by vict-shevchenko** [comment]
Sorry, was away from my laptop. @Maidanskiy thanks a lot for making a PR. You have done a big job on translating and I appreciate a lot. My comments are just to make messages more friendly to a wider audience.

File: src/lang/ua.js (hunk 2 above)
Comment (maidanskyi) [reply src/lang/ua.js:32]: @vict-shevchenko http://slovnyk.ua/index.php?swrd=%E2%E0%EB%B3%E4%ED%E8%E9&x=0&y=0

Comment (vict-shevchenko) [reply src/lang/ua.js:32]: Ok, no more convincing.

**Pull request merged**